### PR TITLE
Added histograms to visualization.

### PR DESCRIPTION
**Pull request by Bentley386**
Also removed visualization from state_graph.py since we have visualization.py for that now.

### CODE REVIEW

File: stream_story_py/visualization.py [hunk 1]
```diff
@@ -70,11 +71,38 @@ def get_TSNE(self, perplexity : int = 100, n_iter : int = 3000) -> go.Figure:
...
                 columns=["tsne_x","tsne_y","cluster"])
         return px.scatter(plot_data,x="tsne_x",y="tsne_y",color="cluster")
+    
+    def get_histograms(self,cluster : int) -> go.Figure:
+        """Returns a figure containing a histogram for each sensor 
+        for the data in a given cluster (cluster index starts at 0).
+        """
+        
+        maxcluster = np.amax(self.labels)
+        if cluster > maxcluster:
```
Comment (aljazkosmerlj) [inline stream_story_py/visualization.py:81]: Not super important, but could just do:
`if cluster not in self.labels:`

File: stream_story_py/visualization.py [hunk 2]
```diff
@@ -70,11 +71,38 @@ def get_TSNE(self, perplexity : int = 100, n_iter : int = 3000) -> go.Figure:
...
         
 if __name__ == "__main__":
     sensor_list = ["50", "53", "55", "62", "63", "64", "65", "97", "98"]
     sensor_values = pd.read_csv(open('../data/B100_hour_SS_input.csv'), index_col=0)
     values = sensor_values.filter(items=["timestamp"] + sensor_list)
-    visual = Visualization(values,5) 
-    plot(visual.get_PCA(),auto_open=True)
\ No newline at end of file
+    visual = Visualization(values,5)
+    #visual.get_histograms(2)
```
Comment (aljazkosmerlj) [inline stream_story_py/visualization.py:107]: Forgotten comment? Probably should be removed.

Comment (aljazkosmerlj) [inline stream_story_py/visualization.py:107]: Forgotten comment? Probably should be removed.

**Review by aljazkosmerlj — CHANGES_REQUESTED** [review]

**Comment by Bentley386** [comment]
Applied the requested changes in that latest commit @aljazkosmerlj

**Pull request merged**